sites/platform/src/create-apps/app-reference/composable-image.md

```diff
@@ -598,6 +598,61 @@ much like you would plug a hard drive into your computer to transfer data.
 
 {{% /note %}}
 
+### Mount types
+
+Each mount type has a specific use case and understanding how to use each type effectively will help optimize your application.
+
+#### `storage` mount
+
+The `storage` mount is a shared network storage that is automatically added by the platform. It is required for applications that need to share files across multiple instances—such as user-uploaded media or shared assets. You can configure the size of this mount via the {{% vendor/name %}} UI.
+
+Pros:
+- Shared across all app and worker instances.
+- Configurable size via the UI.
+- Persistent across restarts and redeployments.
+
+Cons:
+- Lower performance due to network-based access.
```

Comment:
Technically, the `instance` storage is also a network-based volume :sweat_smile: 

The `storage` mount is an NFS volume and not a "block storage" kind of mount, so it has a lot more synchronization overhead. Not sure how to express that in this doc without getting too much into details though...